src/inference/include/openvino/runtime/compiled_model.hpp

```diff
@@ -54,125 +54,125 @@ class OPENVINO_RUNTIME_API CompiledModel {
+     * @brief Gets output of a compiled model identified by @p index.
+     * The output contains information about output tensor such as tensor shape, names, and element type.
+     * @param i Index of input.
+     * @return Compiled model output.
+     * @note The method throws ov::Exception if output with the specified index @p index is not found.
      */
     ov::Output<const ov::Node> output(size_t i) const;
 
     /**
-     * @brief Gets output of a compiled model identified by a @p tensor_name
-     * An output contain information about output tensor such as tensor shape, names and element type
-     * @param tensor_name The output tensor name
-     * @return A compiled model output
-     * @note The method throws ov::Exception if output with specified tensor name @p tensor_name is not found
+     * @brief Gets output of a compiled model identified by @p tensor_name.
+     * The output contains information about output tensor such as tensor shape, names, and element type.
+     * @param tensor_name Output tensor name.
+     * @return Compiled model output.
+     * @note The method throws ov::Exception if output with the specified tensor name @p tensor_name is not found.
      */
     ov::Output<const ov::Node> output(const std::string& tensor_name) const;
 
     /**
      * @brief Creates an inference request object used to infer the compiled model.
-     * The created request has allocated input and output tensors (that can be changed later).
+     * The created request has allocated input and output tensors (which can be changed later).
      *
      * @return InferRequest object
      */
     InferRequest create_infer_request();
 
     /**
      * @brief Exports the current compiled model to an output stream `std::ostream`.
-     * The exported model can also be imported via ov::Core::import_model method
+     * The exported model can also be imported via the ov::Core::import_model method.
      * @see ov::Core::import_model
-     * @param model_stream Output stream to store the model to
+     * @param model_stream Output stream to store the model to.
      */
     void export_model(std::ostream& model_stream);
 
     /**
-     * @brief Sets properties for current compiled model
+     * @brief Sets properties for the current compiled model.
      *
      * @param properties Map of pairs: (property name, property value)
      */
     void set_property(const AnyMap& properties);
 
     /**
-     * @brief Sets properties for current compiled model
+     * @brief Sets properties for the current compiled model.
      *
-     * @tparam Properties Should be the pack of `std::pair<std::string, ov::Any>` types
-     * @param properties Optional pack of pairs: (property name, property value)
-     * @return nothing
+     * @tparam Properties Should be the pack of `std::pair<std::string, ov::Any>` types.
+     * @param properties Optional pack of pairs: (property name, property value).
+     * @return Nothing.
```

Comment:
Should we just remove @return?

Reply:
agree

Reply:
@ilya-lavrenov  @ilyachur Should I remove all similar instances across all headers files?